polkadot/runtime/westend/src/weights/pallet_xcm.rs

```diff
@@ -348,4 +348,11 @@ impl<T: frame_system::Config> pallet_xcm::WeightInfo for WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().reads(1))
 			.saturating_add(T::DbWeight::get().writes(1))
 	}
+
+	fn set_asset_claimer() -> Weight {
+		Weight::from_parts(35_299_000, 0)
+			.saturating_add(Weight::from_parts(0, 3488))
+			.saturating_add(T::DbWeight::get().reads(1))
+			.saturating_add(T::DbWeight::get().writes(1))
+	}
```

Comment:
I believe this is unused/not needed (same for all other runtimes). I see no `set_asset_claimer` fn in `pallet_xcm`.